tss-esapi/src/error/return_code/tpm/format_zero/error.rs

```diff
@@ -5,6 +5,14 @@ use std::convert::TryFrom;
 
 /// Type representing the TPM format zero error
 /// response code.
+///
+/// # Details
+/// The error messages are short forms of the descriptions given in the specification
+/// that describes return codes (see the
+/// [Part 2, Structures](https://trustedcomputinggroup.org/wp-content/uploads/TCG_TPM2_r1p59_Part2_Structures_pub.pdf)).
+/// Sometimes these descriptions refers to fields within the structures described in the specification. When
+/// a message contains such a description then the name of the of the field is put inside single quotation marks
+/// (e.g. 'authValue').
```

Comment:
I think using backticks for `authValue` would be slightly better.

Reply:
Fixed.